src/services/conseillers/controllers/confirmationEmailCandidature.test.ts

```diff
@@ -0,0 +1,73 @@
+import { beforeEach, describe, expect, it } from 'vitest';
+import { viderLesCollections, host, requetePatch } from '../../../tests/utils';
+import { construireConseiller, mailConfirmationAdresseMail } from './creerCandidatureConseiller';
+import app from '../../../app';
+
+describe('Pouvoir confirmer mon adresse mail d’inscription', () => {
+  it(
+    'Quand je valide mon e-mail d’inscription mais que le lien est invalide alors j’ai une erreur',
+    async () => {
+      //GIVEN
+      const tokenInvalide = '1111111111111111111111';

```

Comment:
```suggestion
      const tokenInvalide = '1';
```
Juste comme ça, ça suffirait non ? ^^